tests/queries/0_stateless/02344_describe_cache.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Tags: no-fasttest
```

Comment:
As this is cache test, should we add 'no-parallel' ?

Reply:
No, cache in this test is not shared between tests